src/main/java/core/basesyntax/FigureManager.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class FigureManager {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure randomFigure = figureSupplier.getRandomFigure();
+
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
```

Comment:
```suggestion
        for (int i = 0; i < figures.length / 2; i++) {
            figures[i] = figureSupplier.getRandomFigure();
        }

        for (int i = figures.length / 2; i < figures.length; i++) {
            figures[i] = figureSupplier.getDefaultFigure();
        }
```

create 1 loop for this